datasets/wiki_lingua/README.md

```diff
@@ -0,0 +1,208 @@
+  portuguese:
+  - 10K<n<100K
+  russian:
+  - 10K<n<100K
+  spanish:
+  - 100K<n<500K
+  thai:
+  - 10K<n<100K
+  turkish:
+  - 1K<n<10K
+  vietnamese:
+  - 10K<n<100K
+source_datasets:
+- original
+task_categories:
+- conditional-text-generation
+task_ids:
+- summarization
+---
+# Dataset Card Creation Guide
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-instances)
+  - [Data Splits](#data-instances)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Repository:** [URL](https://github.com/esdurmus/Wikilingua)
+- **Paper:** [WikiLingua: A Multilingual Abstractive Summarization Dataset](https://arxiv.org/abs/2010.03093)
+
+### Dataset Summary
+
+We introduce WikiLingua, a large-scale, multilingual dataset for the evaluation of crosslingual abstractive summarization systems. We extract article and summary pairs in 18 languages from WikiHow, a high quality, collaborative resource of how-to guides on a diverse set of topics written by human authors. We create gold-standard article-summary alignments across languages by aligning the images that are used to describe each how-to step in an article.
+
+### Supported Tasks and Leaderboards
+
+[More Information Needed]
+
+### Languages
+
+The table below shows number of article-summary pairs with a parallel article-summary pair in English. 
+______________________________
+| Language    | Num. parallel |
+| ----------- | --------------|
+| English     |   141,457     |
+| Spanish     |   113,215     |
+| Portuguese  |    81,695     |
+| French      |    63,692     |
+| German      |    58,375     |
+| Russian     |    52,928     |
+| Italian     |    50,968     |
+| Indonesian  |    47,511     |
+| Dutch       |    31,270     |
+| Arabic      |    29,229     |
+| Vietnamese  |    19,600     |
+| Chinese     |    18,887     |
+| Thai        |    14,770     |
+| Japanese    |    12,669     |
+| Korean      |    12,189     |
+| Hindi       |     9,929     |
+| Czech       |     7,200     |
+| Turkish     |     4,503     |
+
+
+## Dataset Structure
+
+### Data Instances
+
+[More Information Needed]
```

Comment:
Can you provide an example from the dataset ? You can get one by doing `load_dataset` and with `dataset["train"][0]` for example

Reply:
Thanks I have added this to the readme